imap_processing/cdf/imap_cdf_manager.py

```diff
@@ -17,8 +17,7 @@ def __init__(self, source_dir=None):
         if source_dir is None:
             super().__init__(Path(__file__).parent / "config")
         else:
-            super().__init__(Path(__file__).parent / "config")
-            self.source_dir = Path(__file__).parent / source_dir
+            super().__init__(Path(__file__).parent / source_dir)
```

Comment:
Actually, could you change this to accept the whole path, rather than assuming the path is a parent to this file?

eg
```
def __init__(self, source_dir: Path=None):
  ...
  else:
    super().__init__(source_dir)
```